tests/data/test_wrappers.py

```diff
@@ -320,191 +313,28 @@ def test_n_transitions_and_empty_error(Env):
         venv.pop_transitions()
 
 
-def _check_obs_or_space_equal(got, expected):
-    assert type(got) is type(expected)
-    if isinstance(got, (Dict, gym.spaces.Dict)):
-        assert len(got.keys()) == len(expected.keys())
-        for k, v in got.items():
-            assert v == expected[k]
-    else:
-        assert got == expected
-
-
 @pytest.mark.parametrize("Env", Envs)
 @pytest.mark.parametrize("original_obs_key", ["k1", "k2"])
 def test_human_readable_wrapper(Env, original_obs_key):
     num_obs_key_expected = 2 if Env is _CountingEnv else 3
     origin_obs_key = original_obs_key if Env is _CountingEnv else "t"
-    env = HumanReadableWrapper(
-        Env(render_mode="rgb_array"),
+    ori_env = Env(render_mode="rgb_array")
+    hr_env = HumanReadableWrapper(
+        ori_env,
         original_obs_key=original_obs_key,
     )
-    expected_hr_space = gym.spaces.Box(
-        low=0,
-        high=255,
-        shape=(10,),
-        dtype=np.uint8,
-    )
-    if Env == _CountingEnv:
-        expected_obs_space = gym.spaces.Dict(
-            {
-                original_obs_key: gym.spaces.Box(low=0, high=np.inf, shape=()),
-                HR_OBS_KEY: expected_hr_space,
-            },
-        )
-    else:
-        expected_obs_space = gym.spaces.Dict(
-            {
-                "2t": gym.spaces.Box(low=0, high=np.inf, shape=()),
-                "t": gym.spaces.Box(low=0, high=np.inf, shape=()),
-                HR_OBS_KEY: expected_hr_space,
-            },
-        )
-    assert isinstance(env.observation_space, gym.spaces.Dict)
-    _check_obs_or_space_equal(env.observation_space, expected_obs_space)
+    assert hr_env.observation_space == ori_env.observation_space
```

Comment:
How can this be true? Does the observation returned by `step()` not belong to the observation space (that'd violate the Gymnasium API I think)? Or are we encoding the human readable information somewhere other than the observation?